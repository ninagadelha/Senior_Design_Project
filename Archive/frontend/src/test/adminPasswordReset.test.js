import renderer from "react-test-renderer";
import React, { useState } from "react";
import AdminPasswordReset from "../routes/adminPasswordReset";

jest.mock("../styles/home.css");
jest.mock("../index.css");
jest.mock("../components/AdminNavBar");
jest.mock("../components/footer");
jest.mock("react-router-dom");
jest.mock("axios");

const renderTree = (tree) => renderer.create(tree);
describe("<AdminPasswordReset>", () => {
  it("should render component", () => {
    expect(renderTree(<AdminPasswordReset />).toJSON()).toMatchSnapshot();
  });
});
