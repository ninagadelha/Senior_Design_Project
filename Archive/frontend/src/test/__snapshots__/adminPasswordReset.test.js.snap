// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdminPasswordReset> should render component 1`] = `
<div
  class="cont"
>
  <div
    class="content-placeholder-results"
  >
    <h1>
      Reset User Password
    </h1>
    <div>
      <label>
        Enter Account Email: 
        <input
          id="emailInput"
          onChange={[Function]}
          type="email"
        />
      </label>
      <br />
      <br />
      <label>
        Enter New Password (minimum of 7 characters): 
        <input
          id="passwordInput"
          onChange={[Function]}
          type="password"
        />
      </label>
      <br />
      <br />
      <label>
        Confirm New Password: 
        <input
          id="confirmInput"
          onChange={[Function]}
          type="password"
        />
      </label>
    </div>
    <div>
      <button
        className="action-button"
        onClick={[Function]}
        style={
          {
            "marginTop": "16px",
          }
        }
      >
        Reset Password
      </button>
    </div>
  </div>
</div>
`;
